### PR TITLE
Use regex to check prefix

### PR DESCRIPTION
**Pull request by bogberry**
Prefix argument is being decoded as a hex string, with a zero appended for odd length strings, when a simple regex check will do

### CODE REVIEW

**Review by alnoki — APPROVED** [review]

**Comment by alnoki** [comment]
@bogberry this is much more straightforward. The only more practical manner I can think of would be something like:

```
    #[arg(short, long, parse(check_str_is_hex)]
    prefix: String,
```

but this would probably involve us implementing the parse function ourselves anyways unless there is a standard parser like this already available.

**Comment by bogberry** [comment]
True, might be a good idea to implement a parser in the future, but for the moment I think I'll leave this as is

**Comment by alnoki** [comment]
@qdrs @elliottdehn

File: src/main.rs [hunk 1]
```diff
@@ -8,9 +8,14 @@ use aptos_types::{
 use chrono::prelude::{DateTime, Local};
 use clap::Parser;
 use crossbeam::channel;
+use once_cell::sync::Lazy;
+use regex::Regex;
 use std::thread::{self, available_parallelism};
 use std::time::{Instant, SystemTime};
 
+/// Regex for checking that prefix only includes valid hex characters.
+static HEX_REGEX: Lazy<Regex> = Lazy::new(|| Regex::new(r"^[0-9a-fA-F]+$").unwrap());
```
Comment (elliottdehn) [inline src/main.rs:17]: seems like a lot of effort importing libraries to make this static. Why not make it run in-line since it only runs once anyway?

**Review by elliottdehn — APPROVED** [review]

**Review by qdrs — APPROVED** [review]

**Pull request merged**